os/tools/compression/Makefile

```diff
@@ -122,3 +118,9 @@ endif
 	@rm -f config.h
 	@rm -f compression.h
 	@rm -f *.h
+	@rm -f .config
+
+distclean: clean
+	$(call DELFILE, Make.dep)
+	$(call DELFILE, .depend)
```

Comment:
To use definition like DELFILE, you must include `Config.mk` by including Make.defs of os folder. But here is no including. Is this really working?

Reply:
I missed to -include $(TOPDIR)/Make.defs, will correct it